client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx

```diff
@@ -0,0 +1,384 @@
+import {
+	iconListCss,
+	listWithDividersCss,
+	whatHappensNextIconCss,
+} from '@/client/styles/GenericStyles';
+import {
+	LoadingState,
+	useAsyncLoader,
+} from '@/client/utilities/hooks/useAsyncLoader';
+import { createProductDetailFetcher } from '@/client/utilities/productUtils';
+import { cancellationFormatDate } from '@/shared/dates';
+import type {
+	MembersDataApiResponse,
+	ProductDetail,
+} from '@/shared/productResponse';
+import { getMainPlan, isProduct } from '@/shared/productResponse';
+import type { ProductType, WithProductType } from '@/shared/productTypes';
+import {
+	calculateMonthlyOrAnnualFromBillingPeriod,
+	GROUPED_PRODUCT_TYPES,
+} from '@/shared/productTypes';
+
+const subHeadingTitleCss = `
+    ${headline.medium()};
+    font-weight: bold;
+    ${until.tablet} {
+      ${headline.xxsmall()};
+      font-weight: bold;
+    };
+  `;
+const subHeadingBorderTopCss = `
+    margin: 16px 0 32px;
+  `;
+
+interface InnerContentProps {
+	manageProductV2Props: WithProductType<ProductType>;
+	productDetail: ProductDetail;
+}
+
+const InnerContent = ({
+	manageProductV2Props,
+	productDetail,
+}: InnerContentProps) => {
+	const mainPlan = getMainPlan(productDetail.subscription);
+	if (!mainPlan) {
+		throw new Error('mainPlan does not exist in manageProductV2 page');
+	}
+
+	const nextPaymentDetails = getNextPaymentDetails(
+		mainPlan,
+		productDetail.subscription,
+		null,
+		!!productDetail.alertText,
+	);
+
+	const specificProductType = manageProductV2Props.productType;
+
+	const groupedProductType =
+		GROUPED_PRODUCT_TYPES[specificProductType.groupedProductType];
+
+	const hasCancellationPending = productDetail.subscription.cancelledAt;
+
+	const cancelledCopy =
+		specificProductType.cancelledCopy || groupedProductType.cancelledCopy;
+
+	console.log(nextPaymentDetails?.paymentInterval);
+
+	const monthlyOrAnnual = calculateMonthlyOrAnnualFromBillingPeriod(
+		nextPaymentDetails?.paymentInterval,
+	).toLowerCase();
+
+	return (
+		<>
+			<PaymentFailureAlertIfApplicable productDetails={[productDetail]} />
+			<div
+				css={css`
+					${subHeadingBorderTopCss};
+					display: flex;
+					align-items: start;
+					justify-content: space-between;
+				`}
+			>
+				<h2
+					css={css`
+						${subHeadingTitleCss};
+						margin: 0;
+					`}
+				>
+					Manage{' '}
+					{specificProductType.productTitle(mainPlan).toLowerCase()}
+				</h2>
+			</div>
+
+			{hasCancellationPending && (
+				<p
+					css={css`
+						${textSans.medium()};
+					`}
+				>
+					<ErrorIcon fill={palette.brandAlt[200]} />
+					<span
+						css={css`
+							margin-left: ${space[2]}px;
+						`}
+					>
+						{cancelledCopy}{' '}
+						<strong>
+							{cancellationFormatDate(
+								productDetail.subscription
+									.cancellationEffectiveDate,
+							)}
+						</strong>
+					</span>
+					.
+				</p>
+			)}
+
+			<BasicProductInfoTable
+				groupedProductType={groupedProductType}
+				productDetail={productDetail}
+			/>
+
+			<h2
```

Comment:
probably should be an `h3` if the Manage bit is an `h2`

Reply:
great spot